app/serializers/person_escort_record_serializer.rb

```diff
@@ -0,0 +1,25 @@
+class PersonEscortRecordSerializer < ActiveModel::Serializer
+  belongs_to :profile, record_type: :profile
+  belongs_to :framework, record_type: :framework
+  has_many :framework_responses, serializer: FrameworkResponseSerializer, key: :responses
+
+  attributes :version, :status
+
+  def status
+    object.state
+  end
+
+  def version
+    object.framework.version
+  end
+
+  def framework_responses
+    object.framework_responses.includes(framework_question: :framework)
```

Comment:
Is there a way to move this to the `PersonEscortRecord` model? It feels slightly strange with the serializer resolving potential performance issues 🤔

Reply:
maybe as a scope? Although it would be somewhat similar just calling on the scope instead. I'm wary of making this the default scope. I tried having a look at optimisations and this seemed like the cleanest way, let me know your thoughts

Reply:
A scope would be slightly nicer - it's really just frustration with AMS I suppose. The way they work under the hood really isn't performant for relationships and it's good to try and address the inevitable N+1 issues. We need to look at a better gem at some point. A scope would be nice but it's not necessary to add right now, particularly if the design for this is still in flux.